spam/01_spam_tutorial.py

```diff
@@ -892,33 +872,17 @@ def plot_probabilities_histogram(Y):
 # %%
 from snorkel.analysis import metric_score
 from snorkel.utils import preds_to_probs
+from utils import get_keras_logreg, get_keras_early_stopping
 
-# %%
 # Our model is a simple linear layer mapping from feature
 # vectors to the number of labels in our problem (2).
-keras_model = tf.keras.Sequential()
-keras_model.add(
-    tf.keras.layers.Dense(
-        units=2,
-        input_dim=X_train.shape[1],
-        activation=tf.nn.softmax,
-        kernel_regularizer=tf.keras.regularizers.l2(0.001),
-    )
-)
-optimizer = tf.keras.optimizers.Adam(lr=0.01)
-keras_model.compile(
-    optimizer=optimizer, loss="categorical_crossentropy", metrics=["accuracy"]
-)
-
-early_stopping = tf.keras.callbacks.EarlyStopping(
-    monitor="val_acc", patience=10, verbose=1, restore_best_weights=True
-)
+keras_model = get_keras_logreg(input_dim=X_train.shape[1])
```

Comment:
What is logreg? I'd just call this get_keras_model or get_keras_mlp

Reply:
(@brahmaneya Logistic regression; I think that abbreviation standard enough to leave in)

@henryre I think this is good to put this stuff in util file but still need to call out that it's *just* standard keras implementation (eg add comment)